packages/cli/test/unit/commands/pipelines/diff.unit.test.ts

```diff
@@ -63,6 +63,8 @@ describe('pipelines:diff', function () {
 
   function mockPipelineCoupling(testInstance: typeof test) {
     return testInstance
+      .stderr()
+      .stdout()
```

Comment:
All changes to this file were to silence hundreds of lines of useless output like:
![image](https://github.com/heroku/cli/assets/6306282/25547544-04ab-41db-9d5b-08438edf8426)